.github/workflows/publish_docker_images_on_tag.yml

```diff
@@ -0,0 +1,47 @@
+name: Build Docker Image On Tag Push
+
+env:
+  DOCKERHUB_USER_NAME: ${{ secrets.DOCKERHUB_USER_NAME }}
+  DOCKERHUB_PASSWORD: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+on:
+  push:
+    tags:
+      - "**"
```

Comment:
should we be more specific here, e.g.: by using "v[0-9]\.[0-9]\.[0-9]" ?

Reply:
I like `\d` more than `[0-9]` that improves readability IMO.

```suggestion
      - "v\d+\.\d+\.\d+"
```

Reply:
"v[0-9]+.[0-9]+.[0-9]+" worked by removing escape characters for dots
\d for digits is not supported for Github actions.
For reference https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
![image](https://user-images.githubusercontent.com/1596331/156526243-92125ffa-aa8a-4227-80c3-a77cab169603.png)

Reply:
👍🏽

I just assumed that any regex would work, and apparently I was wrong.

`v[0-9]+.[0-9]+.[0-9]+` looks good to me